addons/l10n_es/views/report_invoice.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_invoice_document" inherit_id="account.report_invoice_document">
+        <!-- add company id -->
+        <xpath expr="//div[@id='partner_vat_address_not_same_as_shipping']" position="after">
+            <div t-if="o.partner_id.company_registry">
```

Comment:
Oh and shouldn't you add a condition to check that the sender is from Spain? (I think we use `account_fiscal_country_id` in 16+, but I'm not sure)

Reply:
Yeah i was actually thinking the same, i was about to add a new condition

Reply:
I see that most of the time we use the country_code of the company in 16.0 so i decided to go with that, i will export the pot now :+1: